code/modules/medical/surgery_procs.dm

```diff
@@ -1174,68 +1227,6 @@
 		else
 			return 0
 
-
```

Comment:
there are a couple 'heart.op_stage' red sections in the PR. I pulled them as heart had already been converted to chest.op_stage

Reply:
Hah, that's good. When I added the new organs, I just left the heart as-is and didn't much about with it much. I figured it would be better to put it in the check op stage, but didn't really care to go through whatever it would take to make it fit.